projects/aca-content/folder-rules/src/rule-details/conditions/rule-simple-condition.ui-component.html

```diff
@@ -21,14 +21,54 @@
     </mat-select>
   </mat-form-field>
 
-  <mat-form-field class="aca-rule-simple-condition__form__parameter-input">
-    <mat-select formControlName="parameter" data-automation-id="simple-condition-value-select" *ngIf="selectedField?.type === 'mimeType'; else valueInput">
+  <mat-form-field class="aca-rule-simple-condition__form__parameter-input" [ngSwitch]="selectedField.type">
+    <mat-select formControlName="parameter" data-automation-id="simple-condition-value-select" *ngSwitchCase="'mimeType'">
       <mat-option *ngFor="let mimeType of mimeTypes"
         [value]="mimeType.value">
         {{ mimeType.label }}
       </mat-option>
     </mat-select>
-    <ng-template #valueInput>
+    <ng-template [ngSwitchCase]="'autocomplete'">
+        <input
+          matInput
+          [matAutocomplete]="auto"
+          formControlName="parameter"
+          (focusout)="autoSelectValidOption()"
+          data-automation-id="auto-complete-input-field"
```

Comment:
Can we unify spelling? Here you use `auto-complete` and then `autocomplete`

Reply:
The reason why I was using auto-complete is because in the template, its camel-cased as autoComplete. Regardless, I've changed all occurences of autocomplete with auto-complete (except for references to the angular material component)